app/src/test/java/org/simple/clinic/newentry/PatientEntryUpdateTest.kt

```diff
@@ -368,4 +368,30 @@ class PatientEntryUpdateTest {
         )
   }
 
+  @Test
+  fun `when the date of birth exceeds max limit, then show error`() {
+    //given
+    val errors: List<PatientEntryValidationError> = listOf(PatientEntryValidationError.DOB_EXCEEDS_MAX_LIMIT)
+    val givenModel = defaultModel
+        .fullNameChanged("Name")
+        .dateOfBirthChanged("01/02/1800")
```

Comment:
Since the `userInputAgeValidator` takes in a `UserClock` to compare dates, we can expose the intent more clearly in this test by using:

```kotlin
val enteredDate = formatter.format(LocalDate.now(userClock).minusYears(MAX_ALLOWED_PATIENT_AGE))

.dateOfBirthChanged(enteredDate)
```

Reply:
resolved